app/views/schools/_school.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= school.name %></td>
+  <td><%= school.responsable_name %></td>
+  <td><%= school.responsable_contact %></td>
+  <td><%= school.notes %></td>
```

Comment:
Something to consider: if the notes are too long, it will affect how the table is displayed on the page.

One possible solution is to truncate the notes in the index view, but show all of them in the show view...
Maybe something for another PR

Reply:
I created a Trello card for this one ;)